capstone/src/main/webapp/WEB-INF/web.xml

```diff
@@ -51,7 +60,6 @@
     <display-name>Restaurant Servlet</display-name>
     <servlet-name>RestaurantServlet</servlet-name>
     <servlet-class>com.google.step.servlets.RestaurantServlet</servlet-class>
-    <load-on-startup>1</load-on-startup>
```

Comment:
Just to clarify my understanding, does removing this make it so that the servlet container doesn't immediately load once the server starts, but rather when it first receives a request?

Reply:
Oh, sorry. It's kind of a long story, but I'll add it back.